physical/raft/raft.go

```diff
@@ -1009,6 +1041,13 @@ func (b *RaftBackend) applyLog(ctx context.Context, command *LogData) error {
 		return err
 	}
 
+	cmdSize := len(commandBytes)
+	if uint64(cmdSize) > b.maxEntrySize {
+		return fmt.Errorf("%s; got %d bytes, max: %d bytes", physical.ErrValueTooLarge, cmdSize, b.maxEntrySize)
+	}
+
+	defer metrics.AddSample([]string{"raft-storage", "entrySize"}, float32(cmdSize))
```

Comment:
I've seen mixed usage of snake_case, camelCase, and dash-casing. Let me know if there is a preferred method here 👍

Reply:
We've since landed on snake_case.
```suggestion
	defer metrics.AddSample([]string{"raft-storage", "entry_size"}, float32(cmdSize))
```